.github/workflows/docker.yml

```diff
@@ -0,0 +1,25 @@
+name: Docker
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
```

Comment:
question - is there a reason why we build cross-architecture?

Reply:
If you mean https://github.com/docker/build-push-action#multi-platform-image, we don't

Reply:
then why we need QEMU and buildx, aren't they for cross platform docker build?

Reply:
after discussion agree to keep it like this first but once some user complaints we should add platforms